packages/mask/src/plugins/Predict/Azuro/hooks/useMinRate.ts

```diff
@@ -0,0 +1,16 @@
+import { useEffect, useState } from 'react'
+
+export function useMinRate(slippage: number, odd: number) {
+    const [minRate, setMinOdds] = useState<string | number>(odd)
+    const [loading, setLoading] = useState(false)
+
+    useEffect(() => {
```

Comment:
Refactor to use `useAsyncRetry`.

Reply:
`useMinRate` is actually an overkill. Removed.